README.md

```diff
@@ -63,6 +63,11 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
   - `internalConsole` - VS Code debug console (input stream not supported).
   - `integratedTerminal` - VS Code integrated terminal.
   - `externalTerminal` - External terminal that can be configured in user settings.
+- `shortenCommandLine` - The specified command line style to launch the program. Defaults to `auto`.
```

Comment:
Should provide background for this options. In most case, user should not care about this issue. Or add a trouble shooting section for this problem.